app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,7 @@
     <title>GivingProfiles</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
i'm surprised this doesn't get added automatically in rails generate process...

Reply:
So are we ok'ing this does its even work?